op-devstack/sysgo/system.go

```diff
@@ -40,7 +40,7 @@ func NewDefaultInteropSystemIDs(l1ID, l2AID, l2BID eth.ChainID) DefaultInteropSy
 		L1CL:        stack.L1CLNodeID{Key: "l1", ChainID: l1ID},
 		Superchain:  "main", // TODO(#15244): hardcoded to match the deployer default ID
 		Cluster:     "main",
-		Supervisor:  "dev",
+		Supervisor:  "primary",
```

Comment:
The first supervisor was renamed to `primary` and second one as `secondary` to order them alphabetically. To make the matcher work.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/15908/commits/fd196bc16e4fe215c93704398a472df0a21c7b52